### ISSUE TITLE
/categories/ returns no results

### ISSUE THREAD

**Issue by KoesterD** [datenanfragen/website#444]
When I visit https://www.datenanfragen.de/categories/ it shows 2 pages of results, but both are empty.

I expect that page to return the company categories.

This was tested in Chromium 85, Chrome 85, Microsoft Edge 86, Firefox 46 and Firefox 81 with both adblockers active and inactive.

**Comment by KoesterD** [comment]
![image](https://user-images.githubusercontent.com/6953435/95771988-50910b80-0cbc-11eb-820b-55cc4b71bad4.png)
![image](https://user-images.githubusercontent.com/6953435/95772050-69012600-0cbc-11eb-981d-99fc2f6bad48.png)

This is what I see when I visit that page.

**Comment by baltpeter** [comment]
Good catch, thanks! I'll open this for Hacktoberfest if anyone wants to pick it up.

**Comment by Raj-Datta-Manohar** [comment]
Hi, I would like to work on this!

**Comment by baltpeter** [comment]
@Raj-Datta-Manohar Sure, thanks. I'll assign you.

**Comment by kyteinsky** [comment]
I can see this issue persists. I want to resolve this one.
Would you please provide some pointers where I should look first in the codebase?

Thanks.

**Comment by baltpeter** [comment]
@kyteinsky These pages are generated automatically by Hugo. Without having looked into this, this probably happens as we _don't_ have anything for them in our codebase: https://gohugo.io/templates/

**Comment by kyteinsky** [comment]
Hmm, I see.
Is [this](https://github.com/datenanfragen/data/blob/d539a201e45e1ab28aacd765257e546ad67e9117/schema.json#L27) where the categories list is present?
And do I need to bind some link or click action to the individual categories, or is that work already done?
I will look into Hugo files in this repo to find out more.

**Comment by baltpeter** [comment]
The schema defines a list of all the valid categories, yes. But that's not something you'd have to handle. The Hugo taxonomy stuff should take care of that based on the companies that are actually present.

> 

Not sure I understand that question, to be honest. The problem here is likely that our default `summary` templates don't know how to render a `category`. You'd have to create one for that. The category list pages (e.g.: https://www.datarequests.org/categories/school) do already work, as they have a `summary` template already: https://github.com/datenanfragen/website/blob/master/layouts/partials/summary-company.html
